src/js/get-fundraiser.js

```diff
@@ -0,0 +1,62 @@
+const mapping = {
+  14274: { // France
+    url: ' https://www.helloasso.com/associations/rockclimber/formulaires/1/widget'
+  },
+
+  14474: { // Scotland
+    url: 'https://www.7amax.co.uk/'
+  },
+
+  14151: { // Malta
+    url: 'https://maltaclimbingclub.org/bolt-fund/'
+  },
+
+  14125: { // Philippines
+    url: 'http://www.climbphilippines.com/boltfund.html'
+  }
+};
+
+const intersection = function(array1, array2) {
+  return array1.filter(value => array2.includes(value));
+};
+
+export default function(document) {
+  // Only areas, waypoints, routes
+  if (!['r', 'w', 'a'].includes(document.type)) {
+    return null;
+  }
+
+  if (document.type === 'r') {
+    // for routes, only mountain_climbing and rock_climbing activity
+    if (intersection(document.activities, ['mountain_climbing', 'rock_climbing']).length === 0) {
```

Comment:
`if (!intersection(document.activities, ['mountain_climbing', 'rock_climbing']).length)` is enough :)

Reply:
👍